common/script/content/index.coffee

```diff
@@ -1168,11 +1168,19 @@ api.hatchingPotions =
   CottonCandyPink:  value: 4, text: t('hatchingPotionCottonCandyPink')
   CottonCandyBlue:  value: 4, text: t('hatchingPotionCottonCandyBlue')
   Golden:           value: 5, text: t('hatchingPotionGolden')
+
+api.premiumHatchingPotions =
   Spooky:           value: 2, text: t('hatchingPotionSpooky'), addlNotes: t('premiumPotionAddlNotes'), premium: true, limited: true
 
-_.each api.hatchingPotions, (pot,key) ->
+_.each api.dropHatchingPotions, (pot,key) ->
   _.defaults pot, {key, value: 2, notes: t('hatchingPotionNotes', {potText: pot.text}), premium: false, limited: false, canBuy: (()->true)}
 
+_.each api.premiumHatchingPotions, (pot,key) ->
+  _.defaults pot, {key, value: 2, notes: t('hatchingPotionNotes', {potText: pot.text}), premium: true, limited: false, canBuy: (()->true)}
+
+api.hatchingPotions = _.merge(api.dropHatchingPotions, api.premiumPotionAddlNotes)
```

Comment:
I'm guessing autocomplete error due to the `premiumPotionAddlNotes` string, and that's meant to be `api.premiumHatchingPotions` at the end?